Servers/controllers/project.ctrl.ts

```
@@ -265,6 +265,26 @@ export async function createProject(req: Request, res: Response): Promise<any> {
           error: emailError as Error,
         });
       });
+      const actor = await getUserByIdQuery(req.userId!);
+
+      sendSlackNotification(
+        {
+          userId: actor.id!,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)